go.md

```diff
@@ -0,0 +1,24 @@
+# Working with Go language versions in project
```

Comment:
Can be just "Go", then we can have an index of these policies in README.

Reply:
Renamed. Do you want me to add an index in the README?

Reply:
Sure.

Reply:
Added.